doc/citing.rst

```diff
@@ -9,6 +9,8 @@ In general, please cite the software release that you are using. In case you wis
 Software releases
 -----------------
 
+Charl A.P. Linssen, Pooja N. Babu, Levin Schmidt, Tom Tetzlaff, Maryada, Mohamed Ayssar Benelhedi, Jochen M. Eppler, Bernhard Rumpe and Abigail Morrison (2023). **NESTML 6.0.0.** Zenodo. `doi:10.5281/zenodo.8013073 <https://doi.org/10.5281/zenodo.8013073>`_.
```

Comment:
The doi points to version `5.3.0`. This is the correct DOI: https://doi.org/10.5281/zenodo.8380365